ModTek/ModTek.cs

```diff
@@ -718,6 +743,13 @@ internal static IEnumerator<ProgressReport> LoadMoadsLoop()
                     continue;
                 }
 
+                if (modDef.GameVersions.Count > 0 && !modDef.GameVersions.Exists(x => VersionInfo.ProductVersion.StartsWith(x)))
```

Comment:
I'm really bothered by a list of gameversions, why not just a single version for the minimum? I also would call it BattletechMinimumVersion? If a certain mod knows it breaks quickly and requires an exact version, well then also add a BTMaxVersion property, but more than that I don't really think makes sense for the mods I know.

Reply:
That's a good point, we'll do that.